### PR TITLE
Added more Fr translations to the swagger API

### PR DESCRIPTION
**Pull request by DRH-TBS**

### CODE REVIEW

File: fr.css [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+    content: "Mandat";
+}
+
+/*Open Government License - Canada link */
+.swagger-ui .info__license a {
+    visibility: hidden;
+}
+.swagger-ui .info__license a::before {
+    visibility: visible;
+    content: "Licence du gouvernement ouvert - Canada";
```
Comment (wardi) [inline fr.css:16]: this one is specific to our API so we won't be able to use it with ckanext-openapiview

File: fr.css [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+}
+
+/*Live Response - request url*/
+.swagger-ui .responses-wrapper .responses-inner > div > div > div:not(.curl-command) h4 {
+    visibility: hidden;
+    font-size: 0;
+}
+.swagger-ui .responses-wrapper .responses-inner > div > div > div:not(.curl-command) h4::after {
+    visibility: visible;
+    content: "Request URL FR"; /*TODO: <---Translation to FR Needed for: "Request URL"*/
```
Comment (wardi) [inline fr.css]: still waiting on this?

File: openapi.yaml [hunk 3]
```diff
@@ -752,8 +803,12 @@ paths:
                       Conditions de correspondance à sélectionner, demandes de publication de données `application/json` uniquement 
 
                   examples:
-                    (Value not supported for GET requests):
-                      value: ''
+                    en:
+                      (Value not supported for GET requests):
+                        value: ''
+                    fr:
+                      (Valeur non supportée pour les requêtes OGE):
```
Comment (wardi) [inline openapi.yaml]: is "OGE" confusing here because it doesn't match the actual HTTP verb?

**Comment by wardi** [comment]
just a few minor comments otherwise LGTM

File: fr.css (hunk 2 above)
Comment (DRH-TBS) [reply fr.css]: Corrected this in the new commit

File: fr.css (hunk 1 above)
Comment (wardi) [reply fr.css:16]: Why not split the license title in the yaml source instead?

Comment (DRH-TBS) [reply fr.css:16]: Good to know, we won't copy this over to the API extension.

File: openapi.yaml (hunk 3 above)
Comment (DRH-TBS) [reply openapi.yaml]: Good call, changed it back to GET

File: fr.css (hunk 1 above)
Comment (wardi) [reply fr.css:16]: A better fix would be to split the license title in the yaml source.

**Pull request merged**